website/content/api-docs/secret/identity/mfa/duo.mdx

```diff
@@ -0,0 +1,160 @@
+---
+layout: api
+page_title: /identity/mfa/method/duo - HTTP API
+description: >-
+  The '/identity/mfa/method/duo' endpoint focuses on managing Duo MFA behaviors in Vault.
+---
+
+## Configure Duo MFA Method
+
+This endpoint defines an MFA method of type Duo.
+
+| Method | Path                           |
+| :----- |:-------------------------------|
+| `POST` | `/identity/mfa/method/duo/:id` |
+
+### Parameters
+
+- `id` `(string: "")` - Optional UUID to specify if updating an existing method.
+
+- `mount_accessor` `(string: <required>)` - The mount to tie this method to for use in automatic mappings. The mapping will use the Name field of Aliases associated with this mount as the username in the mapping.
```

Comment:
It might be worth calling out that this mount can be in the current or a child namespace, here and on the other methods ( found out as part of testing that the latter is possible, hence figured i'd drop this note )

Reply:
I didn't realize this was the case. I can add a note.

Reply:
https://github.com/hashicorp/vault/pull/14302/commits/7e004097edec2e5c0bdb0e5507ad4272c7d5ba66